json2html/jsonconv.py

```diff
@@ -35,30 +35,21 @@
 
 class Json2Html:
 
-    def convert(self, **kwargs):
+    def convert(self, json="", table_attributes='border="1"'):
         """
             Convert JSON to HTML Table format
         """
 
         # table attributes such as class, id, data-attr-*, etc.
         # eg: table_attributes = 'class = "table table-bordered sortable"'
-        table_attributes = None
-        if 'table_attributes' in kwargs:
-            table_attributes = kwargs['table_attributes']
-        else:
-            # by default HTML table border
-            table_attributes = 'border="1"'
         self.table_init_markup = "<table %s>" % table_attributes
 
-        json_input = None
-        if 'json' in kwargs and kwargs['json']:
-            unvalidated_input = kwargs['json']
-            if type(unvalidated_input) in text_types:
-                json_input = json.loads(unvalidated_input, object_pairs_hook=OrderedDict)
-            else:
-                json_input = unvalidated_input
+        if not json:
+            json_input = {}
+        elif type(json) in text_types:
+            json_input = json_parser.loads(json, object_pairs_hook=OrderedDict)
```

Comment:
Exactly my point. It was working fine as expected.
My point was just to support `{}` and `[]` in the expression[1] as both are considered as valid JSON.

[1]
```
>>> json2html.convert(json = {})
>>> json2html.convert(json = [])
```